export.js

```diff
@@ -61,7 +61,12 @@ _.extend(Avatar, {
 
     // Default text color when displaying the initials.
     // Can also be set to a function to map an user object to a text color.
-    textColor: "#fff"
+    textColor: "#fff",
+
+    // Default CSS included in the head of your application.
+    // Setting this to false, or 0, will exclude the default CSS and leave the 
+    // avatar unstyled by the package.
+    includeDefaultCSS: 1
```

Comment:
Please rename this to `includeGeneratedCSS`.

The CSS isn't just a set of default values, it's generated according to the options in `Avatar`. I don't wish to give new users the wrong idea.